ecosystem.md

```diff
@@ -1,106 +1,5 @@
 ---
 title: "Ecosystem"
----
-<br>
-Apache Flink supports a broad ecosystem and works seamlessly with
```

Comment:
Why not keep this page and frontpage link, and just update it to point to flink-packages.org + explanation as to what it is?
Are we just trying to reduce overhead since f-p.o already contains a description on the main page?

Reply:
I also thought about introducing an intermediate page, but users would have to kick through one level of indirection to get to the page.
I fear that people will not find the link to flink-package.org.